Cannot run when downloaded via NPM

For some reason when i try to run any of the commands with a clean npm install i get this error when attempting to import one of the translation files. 

ember-i18n-csv to-csv --locales-path=[mypath] --csv-path=i18n.csv

```
try{
   obj = require(filePath).default;
 }catch(ex){
    console.log(ex);
 }
```
`[SyntaxError: Unexpected reserved word]`

I seems like node is not happy with my ES6 style export statements. 
It doesn't seem to matter that `require('babel-register');` is present. 
```
node: v4.4.5
npm: 3.5.3
```
```
ember-i18n-csv@0.1.0 C:\Projects\GIT\my-app\node_modules\ember-i18n-csv
├─┬ babel-register@6.9.0
│ ├─┬ babel-core@6.9.1
│ │ ├─┬ babel-code-frame@6.8.0
│ │ │ ├─┬ chalk@1.1.3
│ │ │ │ ├── ansi-styles@2.2.1
│ │ │ │ ├── escape-string-regexp@1.0.5
│ │ │ │ ├── has-ansi@2.0.0
│ │ │ │ └── supports-color@2.0.0
│ │ │ ├── esutils@2.0.2
│ │ │ └── js-tokens@1.0.3
│ │ ├─┬ babel-generator@6.9.0
│ │ │ ├─┬ detect-indent@3.0.1
│ │ │ │ ├── get-stdin@4.0.1
│ │ │ │ ├── minimist@1.2.0
│ │ │ │ └─┬ repeating@1.1.3
│ │ │ │   └── is-finite@1.0.1
│ │ │ └── lodash@4.13.1
│ │ ├── babel-helpers@6.8.0
│ │ ├── babel-messages@6.8.0
│ │ ├─┬ babel-template@6.9.0
│ │ │ └── lodash@4.13.1
│ │ ├─┬ babel-traverse@6.9.0
│ │ │ ├── globals@8.18.0
│ │ │ ├─┬ invariant@2.2.1
│ │ │ │ └── loose-envify@1.2.0
│ │ │ └── lodash@4.13.1
│ │ ├─┬ babel-types@6.9.1
│ │ │ ├── lodash@4.13.1
│ │ │ └── to-fast-properties@1.0.2
│ │ ├── babylon@6.8.0
│ │ ├── convert-source-map@1.2.0
│ │ ├─┬ debug@2.2.0
│ │ │ └── ms@0.7.1
│ │ ├── json5@0.4.0
│ │ ├── lodash@4.13.1
│ │ ├─┬ minimatch@2.0.10
│ │ │ └─┬ brace-expansion@1.1.4
│ │ │   ├── balanced-match@0.4.1
│ │ │   └── concat-map@0.0.1
│ │ ├── path-is-absolute@1.0.0
│ │ ├── private@0.1.6
│ │ ├── shebang-regex@1.0.0
│ │ ├── slash@1.0.0
│ │ └── source-map@0.5.6
│ ├─┬ babel-runtime@6.9.2
│ │ └── regenerator-runtime@0.9.5
│ ├── core-js@2.4.0
│ ├─┬ home-or-tmp@1.0.0
│ │ ├── os-tmpdir@1.0.1
│ │ └── user-home@1.1.1
│ ├── lodash@4.13.1
│ ├─┬ mkdirp@0.5.1
│ │ └── minimist@0.0.8
│ ├── path-exists@1.0.0
│ └─┬ source-map-support@0.2.10
│   └─┬ source-map@0.1.32
│     └── amdefine@1.0.0
├── csv-parse@1.1.0
├─┬ csv-stringify@1.0.4
│ └─┬ lodash.get@4.3.0
│   └─┬ lodash._stringtopath@4.8.0
│     └── lodash._basetostring@4.12.0
├── UNMET DEPENDENCY denodeify@^1.0.0
├── UNMET DEPENDENCY eol@0.2.0
├── UNMET DEPENDENCY lodash@^4.0.0
├── UNMET DEPENDENCY promise@^7.0.0
└─┬ yargs@4.7.1
  ├── camelcase@3.0.0
  ├─┬ cliui@3.2.0
  │ ├─┬ strip-ansi@3.0.1
  │ │ └── ansi-regex@2.0.0
  │ └── wrap-ansi@2.0.0
  ├── decamelize@1.2.0
  ├─┬ lodash.assign@4.0.9
  │ ├── lodash.keys@4.0.7
  │ └── lodash.rest@4.0.3
  ├─┬ os-locale@1.4.0
  │ └─┬ lcid@1.0.0
  │   └── invert-kv@1.0.0
  ├─┬ pkg-conf@1.1.3
  │ ├─┬ find-up@1.1.2
  │ │ ├── path-exists@2.1.0
  │ │ └─┬ pinkie-promise@2.0.1
  │ │   └── pinkie@2.0.4
  │ ├─┬ load-json-file@1.1.0
  │ │ ├── graceful-fs@4.1.4
  │ │ ├─┬ parse-json@2.2.0
  │ │ │ └─┬ error-ex@1.3.0
  │ │ │   └── is-arrayish@0.2.1
  │ │ ├── pify@2.3.0
  │ │ └─┬ strip-bom@2.0.0
  │ │   └── is-utf8@0.2.1
  │ ├── object-assign@4.1.0
  │ └── symbol@0.2.3
  ├─┬ read-pkg-up@1.0.1
  │ └─┬ read-pkg@1.1.0
  │   ├─┬ normalize-package-data@2.3.5
  │   │ ├── hosted-git-info@2.1.5
  │   │ ├─┬ is-builtin-module@1.0.0
  │   │ │ └── builtin-modules@1.1.1
  │   │ ├── semver@5.1.0
  │   │ └─┬ validate-npm-package-license@3.0.1
  │   │   ├─┬ spdx-correct@1.0.2
  │   │   │ └── spdx-license-ids@1.2.1
  │   │   └─┬ spdx-expression-parse@1.0.2
  │   │     └── spdx-exceptions@1.0.4
  │   └── path-type@1.1.0
  ├── require-main-filename@1.0.1
  ├── set-blocking@1.0.0
  ├─┬ string-width@1.0.1
  │ ├─┬ code-point-at@1.0.0
  │ │ └── number-is-nan@1.0.0
  │ └── is-fullwidth-code-point@1.0.0
  ├── window-size@0.2.0
  ├── y18n@3.2.1
  └─┬ yargs-parser@2.4.0
    └── camelcase@2.1.1

npm ERR! missing: denodeify@^1.0.0, required by ember-i18n-csv@0.1.0
npm ERR! missing: eol@0.2.0, required by ember-i18n-csv@0.1.0
npm ERR! missing: lodash@^4.0.0, required by ember-i18n-csv@0.1.0
npm ERR! missing: promise@^7.0.0, required by ember-i18n-csv@0.1.0
```

--- a/lib/to-csv.js
+++ b/lib/to-csv.js
@@ -4,10 +4,7 @@ import csvStringify from 'csv-stringify';
 import eol from 'eol';
 import denodeify from 'denodeify';
 import map from 'lodash/map';
-
-// this is needed to `require` the js locale files
-// instead of using fs.read
-import 'babel-register';
+import parseJsonFromJsFile from './utils/parse-json-from-js-file';
 
 const readdir = denodeify(fs.readdir);
 const stat = denodeify(fs.stat);
@@ -73,8 +70,7 @@ export default function(localesPath, csvPath, { onlyMissing } = {}) {
     for (let columnIndex in filePaths) {
       let filePath = filePaths[columnIndex];
 
-      // this is why we need `babel-register` above
-      let json = require(filePath).default;
+      let json = parseJsonFromJsFile(filePath);
 
       recurse(json, columnIndex, 0, '');
     }

--- a/lib/to-js.js
+++ b/lib/to-js.js
@@ -4,6 +4,7 @@ import csvParse from 'csv-parse';
 import eol from 'eol';
 import denodeify from 'denodeify';
 import Promise from 'promise';
+import parseJsonFromJsFile from './utils/parse-json-from-js-file';
 
 const readFile = denodeify(fs.readFile);
 const writeFile = denodeify(fs.writeFile);
@@ -22,7 +23,7 @@ export default function(csvPath, localesPath, { ignoreJshint, merge } = {}) {
       if (merge) {
         let filePath = path.resolve(localesPath, locales[i], 'translations.js');
         if (fs.existsSync(filePath)) {
-          obj = require(filePath).default;
+          obj = parseJsonFromJsFile(filePath);
         }
       }
       objs.push(obj);

--- a/lib/utils/parse-json-from-js-file.js
+++ b/lib/utils/parse-json-from-js-file.js
@@ -1,0 +1,8 @@
+import fs from 'fs';
+
+export default filePath => {
+  let js = fs.readFileSync(filePath, 'utf8');
+  let jsonString = js.substring(js.indexOf('{'), js.lastIndexOf('}') + 1);
+  let json = JSON.parse(jsonString);
+  return json;
+};

--- a/package.json
+++ b/package.json
@@ -28,7 +28,6 @@
   },
   "homepage": "https://github.com/kellyselden/ember-i18n-csv#readme",
   "dependencies": {
-    "babel-register": "^6.0.0",
     "csv-parse": "^1.0.0",
     "csv-stringify": "^1.0.0",
     "denodeify": "^1.0.0",
@@ -41,6 +40,7 @@
     "babel-cli": "^6.0.0",
     "babel-preset-es2015": "^6.0.0",
     "babel-preset-stage-2": "^6.0.0",
+    "babel-register": "^6.0.0",
     "chai": "^3.0.0",
     "coveralls": "^2.0.0",
     "eslint": "^2.0.0",

--- a/test/fixtures/locales-only-missing/nl-nl/translations.js
+++ b/test/fixtures/locales-only-missing/nl-nl/translations.js
@@ -14,7 +14,7 @@ export default {
       "node2-2-1": "test",
       "node2-2-2": "test"
     }
-  },
+  }
 };
 
 /* jshint ignore:end */

--- a/test/fixtures/locales-only-missing/pt-br/translations.js
+++ b/test/fixtures/locales-only-missing/pt-br/translations.js
@@ -10,7 +10,7 @@ export default {
   },
   "node2": {
     "node2-2": {
-      "node2-2-1": "test",
+      "node2-2-1": "test"
     }
   },
   "weekday": {

